backend/src/main/java/com/crm/model/db/CarType.java

```diff
@@ -0,0 +1,13 @@
+package com.crm.model.db;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity(name = "car_type")
+public class CarType {
```

Comment:
As mentioned above, should be named CarTypeEntity to be in line with CarEntity, or maybe better to just leave CarType and CarEntity change to Car

Reply:
done